lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_remove_testers_action.rb

```diff
@@ -0,0 +1,79 @@
+require 'fastlane/action'
+require 'fastlane_core/ui/ui'
+
+require_relative '../helper/firebase_app_distribution_helper'
+require_relative '../helper/firebase_app_distribution_auth_client'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionRemoveTestersAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        if blank?(params[:emails]) && blank?(params[:file])
+          UI.user_error!("Must specify `emails` or `file`.")
+        end
+
+        emails = get_value_from_value_or_file(params[:emails], params[:file]).split(',')
+
+        if emails.count > 1000
+          UI.user_error!("A maximum of 1000 testers can be removed at a time.")
+        end
+
+        count = fad_api_client.remove_testers(params[:project], emails)
+
+        UI.success("✅ #{count} tester(s) removed successfully.")
+      end
+
+      def self.description
+        "Delete testers"
+      end
+
+      def self.authors
+        ["Tunde Agboola"]
+      end
+
+      # supports markdown.
+      def self.details
+        "delete testers"
```

Comment:
Same here - more details? (Also capitalization on this one)

Reply:
Done.

Reply:
Same thing here as above, should update `self.details`

Reply:
Ahh my bad. Updated `details` as well.